eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -49,6 +53,8 @@ if [ "$STAGE" = "end2end" ]; then
    run_e2e_test '' 'cd node_tests && npm run test_operator && npm run test_ui'
 elif [ "$STAGE" = "debug" ]; then
    run_e2e_test '-ti' 'bash'
+elif [ "$STAGE" = "smoke" ]; then
+   run_smoke_test '' 'kubectl get secret -n zenko && echo ${ADMIN_ACCESS_KEY_ID} && echo ${ADMIN_SECRET_ACCESS_KEY} && echo ${$(kubectl get secret)} && cd node_tests && npm run test_smoke'
```

Comment:
use the default namespace
```suggestion
   run_smoke_test '' 'kubectl get secret && echo ${ADMIN_ACCESS_KEY_ID} && echo ${ADMIN_SECRET_ACCESS_KEY} && echo ${$(kubectl get secret)} && cd node_tests && npm run test_smoke'
```

Reply:
same how do I find the right name of the entities?

Reply:
I don't have the answer to this question, maybe @alexanderchan-scality have one.

Reply:
I found them :) 
Thanks